### ISSUE TITLE
Project still active?

### ISSUE THREAD

**Issue by amilajack** [guerrerocarlos/chromecast-js#33]
Is this project still active?

**Comment by Fenny** [comment]
I don't think so, I created my own wrapper for the browser if that could help you!
https://github.com/Fenny/ChromecastJS
